pkgs/development/guile-modules/guile-gcrypt/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchurl
+, guile
+, libgcrypt
+, autoreconfHook
+, pkgconfig
+, texinfo
+}:
+
+stdenv.mkDerivation rec {
+  pname = "guile-gcrypt";
+  version = "0.2.1";
+
+  src = fetchurl {
```

Comment:
Can we use fetchFromGitea instead?

Reply:
I'm not sure. From looking at the interface I think it's Gogs instead. I know Gitea is a Gogs fork but would they be compatible?

Reply:
I'm not sure. From looking at the interface I think it's Gogs instead. I know Gitea is a Gogs fork but would they be compatible?

Reply:
most likely